src/Aspire.Dashboard/DashboardWebApplication.cs

```diff
@@ -336,6 +334,19 @@ public Func<EndpointInfo> OtlpServiceHttpEndPointAccessor
         }
     }
 
+    internal static List<string> GetSupportedCultures()
+    {
+        // our localization list comes from https://github.com/dotnet/arcade/blob/89008f339a79931cc49c739e9dbc1a27c608b379/src/Microsoft.DotNet.XliffTasks/build/Microsoft.DotNet.XliffTasks.props#L22
```

Comment:
Feels to me like this comment would be better alongside the `LocalizedCultures` property above.